compute/instances/creating-instance-with-custom-machine-type/create_instance_with_custom_machine_type_without_helper.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_custom_machine_type_create_without_helper]
+import (
+	"fmt"
+	"io"
+)
+
+// createInstanceWithCustomMachineTypeWithoutHelper create new VM instances without using a CustomMachineType struct.
+func createInstanceWithCustomMachineTypeWithoutHelper(w io.Writer, projectID, zone, instanceName string, coreCount, memory int) [](error) {
```

Comment:
I'm not sure what your purpose is here. What are these calls intended to demonstrate?
to me, this looks more like a test case than a snippet. perhaps it belongs there instead?